src/inspector_socket.cc

```diff
@@ -48,16 +49,17 @@ static void dump_hex(const char* buf, size_t len) {
 }
 #endif
 
+static void remove_from_beginning(std::vector<char>* buffer, size_t count) {
```

Comment:
The house style in node.js is pointers when arguments are modified.  References should always be const.